lang/lang_main.php

```diff
@@ -9,30 +9,42 @@
  * @version 3.3.0
  */
 
+function load_lang_file(&$lang, $ui_lang, $filename, $parent_dir = ".") {
```

Comment:
IIUC, $lang is a global variable, but I need it as a reference variable for the function.

Otherwise `require_once` will consider $lang as local variable and translations will not load.